src/display-pokemon-list.js

```diff
@@ -0,0 +1,281 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-unused-vars */
+/* eslint-disable array-callback-return */
+/* eslint-disable consistent-return */
+/* eslint-disable no-use-before-define */
+
```

Comment:
- [ ] Same as here

Reply:
This issue is addressed. Great work! :+1: